### ISSUE TITLE
mixed-line-ending -- KeyError: b'\n'

### ISSUE THREAD

**Issue by dongyuzheng** [pre-commit/pre-commit-hooks#239]
`pre-commit 1.1.2`

```bash
Mixed line ending........................................................Failed
hookid: mixed-line-ending

Traceback (most recent call last):
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/bin/mixed-line-ending", line 11, in <module>
    load_entry_point('pre-commit-hooks==0.9.4', 'console_scripts', 'mixed-line-ending')()
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/lib/python3.6/site-packages/pre_commit_hooks/mixed_line_ending.py", line 78, in main
    retv |= fix_filename(filename, args.fix)
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/lib/python3.6/site-packages/pre_commit_hooks/mixed_line_ending.py", line 58, in fix_filename
    del counts[target_ending]
KeyError: b'\n'
```
```yaml
-   repo: git://github.com/pre-commit/pre-commit-hooks
    sha: v0.9.4
    hooks:
    -   id: mixed-line-ending
        args: [--fix=lf]
```

**Comment by asottile** [comment]
Thanks for the issue! If possible can you find which file it is that would help greatly in getting to a fix :)

Probably something like `git ls-files --... | xargs -I{} bash -c 'pre-commit run --files {} || echo {}' `

Apologies if the command doesn't work, on phone :)

**Comment by dongyuzheng** [comment]
You mean any file which gives this error?

https://github.com/gymapplife/backend/blob/master/backend/manage.py

```bash
(venv) gary@gary-VirtualBox:~/se464/backend$ pre-commit run --files backend/manage.py 
Reorder python imports...................................................Passed
pyupgrade................................................................Passed
Add trailing commas......................................................Passed
Don't commit to branch...................................................Failed
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
autopep8 wrapper.........................................................Passed
Fix double quoted strings................................................Passed
Fix python encoding pragma...............................................Passed
Debug Statements (Python)................................................Passed
Fix requirements.txt.................................(no files to check)Skipped
Flake8...............................................(no files to check)Skipped
Mixed line ending........................................................Failed
hookid: mixed-line-ending

Traceback (most recent call last):
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/bin/mixed-line-ending", line 11, in <module>
    load_entry_point('pre-commit-hooks==0.9.4', 'console_scripts', 'mixed-line-ending')()
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/lib/python3.6/site-packages/pre_commit_hooks/mixed_line_ending.py", line 78, in main
    retv |= fix_filename(filename, args.fix)
  File "/home/gary/.cache/pre-commit/repos71slzol/py_env-python3.6/lib/python3.6/site-packages/pre_commit_hooks/mixed_line_ending.py", line 58, in fix_filename
    del counts[target_ending]
KeyError: b'\n'
```

**Comment by asottile** [comment]
Awesome! I'll look at this when I'm at a computer!

**Comment by asottile** [comment]
Thanks again for the great issue!  This is now available in v0.9.5!
